tests/recipes/SupportedBrowser.html

```diff
@@ -0,0 +1,4 @@
+<html> 
```

Comment:
I don't see any changes to the tests including false positives or the negation of what you're looking for.